src/ui/theme/designSystemTypographie.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import { ComponentStory } from '@storybook/react'
+import React from 'react'
+
+import { Spacer } from 'ui/components/spacer/Spacer'
+
+import { TypoDS } from './designSystemTypographie'
+
+export default {
+  title: 'Fondations/TypoDS',
+}
+
+export const Typographies: ComponentStory<typeof TypoDS.Title1> = () => (
+  <React.Fragment>
+    <Spacer.Column numberOfSpaces={2} />
```

Comment:
dégageons tout ces espaces superflux

```suggestion
  <ViewGap gap={2} >
```